src/applications/veteran-id-card/config.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+module.exports = {
+
+  messages: {
+    VIC002: "We're sorry. We're having trouble finding your records in our system right now. Please go to [AccessVA] to request a Veteran ID Card.",
+    VIC003: "We're sorry. We can't proceed with your request for a Veteran ID card because we can't confirm your eligibility right now. Please call the Vets.gov Help Desk at 1-855-574-7286, TTY: 1-800-877-8339, Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. (ET).",
+    VIC010: "We're sorry. We can't proceed with your request for a Veteran ID card because we can't confirm your military history right now. Please try again in a few minutes. If it still doesn't work, please call the Vets.gov Help Desk at 1-855-574-7286, TTY: 1-800-877-8339, Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. (ET).",
+    VIC011: "We're sorry. We can't proceed with your request for a Veteran ID card because we can't confirm your military history right now. Please try again in a few minutes. If it still doesn't work, please call the Vets.gov Help Desk at 1-855-574-7286, TTY: 1-800-877-8339, Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. (ET).",
+    'default': "We're sorry, but something went wrong. Please try again in a few moments.",
+    VICV5: "We're sorry. Something went wrong on our end. We can't process your request right now. Please go to [AccessVA] to request a Veteran ID Card.",
+    VICV2: (<p>We’re sorry. We can’t confirm your eligibility for a Veteran ID card right now. To qualify for a Veteran ID Card, both of these must be true:
+      <ul>
```

Comment:
This content update required an unordered list. I (think) just changing this file to `.jsx` and importing `React` is all I need in order for this to support jsx partials.